vllm/entrypoints/llm.py

```diff
@@ -283,7 +283,8 @@ def generate(
         lora_request: Optional[Union[List[LoRARequest], LoRARequest]] = None,
         prompt_adapter_request: Optional[PromptAdapterRequest] = None,
         guided_options_request: Optional[Union[LLMGuidedOptions,
-                                               GuidedDecodingRequest]] = None
+                                               GuidedDecodingRequest]] = None,
+        priority: Optional[List[int]] = None,
```

Comment:
Add this to the docstring below, mention only applicable when `"priority"` scheduling mode is enabled.